### ISSUE TITLE
Add flowcast package as an example

### ISSUE THREAD

**Issue by katharinebreeze** [moj-analytical-services/writing_functions_in_r#46]
https://github.com/moj-analytical-services/prisonflowcast

**Comment by markrangan** [comment]
Sounds like a good idea. We can talk briefly about this in the 2nd half of the course.

**Comment by katharinebreeze** [comment]
yeah there's a part of the course where we list examples so could add it there. though actually now I'm thinking about it we might not be able to include it as it's not public (and this repo is). if we want to we should check with Matt Funnell.
